### ISSUE TITLE
[Bug]: The command 'downloadS3' hasn't supported absolute path yet

### ISSUE THREAD

**Issue by NicoYuan1986** [towhee-io/towhee#2294]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The command 'downloadS3' hasn't supported absolute path yet.
```
towhee downloadS3 -pl ./ -pb s3://pretrainedweights.towhee.io/test/test.pt
Traceback (most recent call last):
  File "/Users/zilliz/virtual-environment/towhee3.9/bin/towhee", line 8, in <module>
    sys.exit(main())
  File "/Users/zilliz/virtual-environment/towhee3.9/lib/python3.9/site-packages/towhee/command/cmdline.py", line 54, in main
    main_body(sys.argv[1:])
  File "/Users/zilliz/virtual-environment/towhee3.9/lib/python3.9/site-packages/towhee/command/cmdline.py", line 50, in main_body
    actions[parsed.action](parsed)()
  File "/Users/zilliz/virtual-environment/towhee3.9/lib/python3.9/site-packages/towhee/command/execute.py", line 203, in __call__
    if not s3.download_files(self._args.pathbucket, self._args.pathlocal):
  File "/Users/zilliz/virtual-environment/towhee3.9/lib/python3.9/site-packages/towhee/command/s3.py", line 159, in download_files
    list_content = self.s3.list_objects_v2(Bucket=self.bucket_name, Prefix=path_prefix)['Contents']
KeyError: 'Contents'
```

### Expected Behavior

succeed

### Steps To Reproduce

```markdown
towhee downloadS3 -pl ./ -pb s3://pretrainedweights.towhee.io/test/test.pt
```

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93): 0.8.2.dev83
- OS(Ubuntu or CentOS):
- CPU/Memory:
- GPU:
- Others:
```

**Comment by NicoYuan1986** [comment]
Now that 'lsS3' has supported this way, can we apply it on 'downloadS3' too?
What do you think？ @zhujiming

**Comment by zhujiming** [comment]
it was fixed #2297

**Comment by NicoYuan1986** [comment]
The update error message:
```
path s3://pretrainedweights.towhee.io/test/test.pt is not exist. please check path[name_space]/[operator_name]/[version]/[file]
download files from s3 error, please check command.
```
Fixed.
